src/Resolver.js

```diff
@@ -27,7 +27,7 @@ class Resolver {
   }
 
   async resolve(input, parent) {
-    let filename = input;
+    let filename = decodeURIComponent(input);
```

Comment:
I don't think this is the right place to do this. We should only decode when we know that the input is a URL. For example, you could legitimately have a filename called "my%20file" where %20 is not a space.

I think we should do this in `addURLDependency` where we know the input is supposed to be a URL. Note that we already handled this for HTML assets, so we should move that call into `addURLDependency` itself so that it applies to e.g. CSS as well. https://github.com/parcel-bundler/parcel/blob/master/src/assets/HTMLAsset.js#L90

Reply:
Fixed it in the latest commit, now it supports url encoded files from any asset, and doesn't pass url encoded strings to asset.dependencies